composer.json

```diff
@@ -32,13 +32,17 @@
         "contao/manager-bundle": "4.9.* <4.9.4"
     },
     "extra": {
+        "bamarni-bin": {
+            "bin-links": false,
+            "target-directory": "tools"
+        },
         "class": [
             "Contao\\ManagerPlugin\\Composer\\ArtifactsPlugin",
             "Contao\\ManagerPlugin\\Composer\\ManagerPluginInstaller",
             "Contao\\ManagerPlugin\\Composer\\AppAutoloadPlugin"
         ],
         "symfony": {
-            "require": "3.4.* || 4.4.* || 5.1.*"
```

Comment:
I think we can drop this with Composer 2.2?

Reply:
Removed in da43df932f293e59a2333aa4968469fb0334e9d0.